scene/resources/capsule_shape_2d.cpp

```diff
@@ -72,6 +72,7 @@ real_t CapsuleShape2D::get_radius() const {
 void CapsuleShape2D::set_height(real_t p_height) {
 
 	height = p_height;
+	if (height < 0) height = 0;
```

Comment:
Please don't put condition and statement on the same line.

Reply:
Updated.